<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tela_principal</class>
 <widget class="QMainWindow" name="tela_principal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1162</width>
    <height>752</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Simulador</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget_principal">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>-1</y>
      <width>1141</width>
      <height>721</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="label_abertura">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>300</y>
        <width>271</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Simulador - Redes Ópticas</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_abertura_cont">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>330</y>
        <width>161</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Elásticas - UFCG</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>160</y>
        <width>164</width>
        <height>341</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="pushButton_nova_sim">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Nova Simulação</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_abrir_sim">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Abrir Simulação</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_exemplos">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Exemplos</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_sobre">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Sobre</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLabel" name="label_op_sim">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>50</y>
        <width>141</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Escolha a simulação:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_op_sim">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>50</y>
        <width>201</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_op_topo">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>110</y>
        <width>141</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Escolha a topologia:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_op_topo">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>110</y>
        <width>201</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_num_nos">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>170</y>
        <width>121</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Número de Nós:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_num_nos">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>170</y>
        <width>113</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_op_osnr_sim_nao">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>230</y>
        <width>211</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>A OSNR deve ser considerada?</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_op_osnr_sim">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>230</y>
        <width>117</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Sim</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup_op_osnr_sim_nao</string>
      </attribute>
     </widget>
     <widget class="QRadioButton" name="radioButton_op_osnr_nao">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>230</y>
        <width>117</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Não</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup_op_osnr_sim_nao</string>
      </attribute>
     </widget>
     <widget class="QLabel" name="label_valor_banda_slot">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>290</y>
        <width>381</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Entre com a banda de um slot, em GHz (valores comuns</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_valor_banda_slot_cont">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>310</y>
        <width>341</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>são 1.5625, 3.125, 6.25 e 12.5):</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_valor_banda_slot">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>310</y>
        <width>131</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_num_slots_enlace">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>370</y>
        <width>201</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Número de slots por enlace:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_num_slots_enlace">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>370</y>
        <width>113</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_op_alg_aloc">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>440</y>
        <width>251</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Entre com o algoritmo de alocação:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_op_alg_aloc">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>440</y>
        <width>111</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_mu">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>510</y>
        <width>351</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Entre com o mu (taxa de desativação de conexões): </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_mu">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>530</y>
        <width>113</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_cons_exp_compr">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>590</y>
        <width>361</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Considerar expansão e compressão do número </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_cons_exp_compr_cont">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>610</y>
        <width>271</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>de subportadoras das conexões? </string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_cons_exp_compr_sim">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>630</y>
        <width>117</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Sim</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup_cons_exp_compr</string>
      </attribute>
     </widget>
     <widget class="QRadioButton" name="radioButton_cons_exp_compr">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>630</y>
        <width>117</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Não</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup_cons_exp_compr</string>
      </attribute>
     </widget>
     <widget class="QPushButton" name="pushButton_proximo">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>690</y>
        <width>99</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Próximo</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_anterior">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>690</y>
        <width>99</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Anterior</string>
      </property>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>473</x>
        <y>30</y>
        <width>20</width>
        <height>611</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QStackedWidget" name="stackedWidget_parte2">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>30</y>
        <width>451</width>
        <height>611</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page_3"/>
      <widget class="QWidget" name="page_4"/>
      <widget class="QWidget" name="page_5"/>
      <widget class="QWidget" name="page_6"/>
      <widget class="QWidget" name="page_7"/>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1162</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArquivo">
    <property name="title">
     <string>Arquivo</string>
    </property>
    <addaction name="actionNova_Simula_o"/>
    <addaction name="actionAbrir_Simula_o"/>
    <addaction name="actionSalvar_Simula_o"/>
    <addaction name="actionExemplos"/>
    <addaction name="actionSair"/>
   </widget>
   <widget class="QMenu" name="menuEditar">
    <property name="title">
     <string>Editar</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuAjuda">
    <property name="title">
     <string>Ajuda</string>
    </property>
    <addaction name="actionSobre"/>
   </widget>
   <addaction name="menuArquivo"/>
   <addaction name="menuEditar"/>
   <addaction name="menuAjuda"/>
  </widget>
  <action name="actionNova_Simula_o">
   <property name="text">
    <string>Nova Simulação</string>
   </property>
  </action>
  <action name="actionAbrir_Simula_o">
   <property name="text">
    <string>Abrir Simulação</string>
   </property>
  </action>
  <action name="actionSalvar_Simula_o">
   <property name="text">
    <string>Salvar Simulação</string>
   </property>
  </action>
  <action name="actionSair">
   <property name="text">
    <string>Sair</string>
   </property>
  </action>
  <action name="actionSobre">
   <property name="text">
    <string>Sobre</string>
   </property>
  </action>
  <action name="actionExemplos">
   <property name="text">
    <string>Exemplos</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup_op_osnr_sim_nao"/>
  <buttongroup name="buttonGroup_cons_exp_compr"/>
 </buttongroups>
</ui>
